lib/fastlane/plugin/wpmreleasetoolkit/actions/common/create_release_action.rb

```diff
@@ -83,6 +83,12 @@ def self.available_options
                                        optional: true,
                                        default_value: false,
                                        is_string: false),
+          FastlaneCore::ConfigItem.new(key: :is_draft,
```

Comment:
You added that parameter (aka `ConfigItem`) to the Fastlane action… but didn't make use of it in the implementation (inside the `self.run` method of the `CreateReleaseAction` class) when actually calling the helper method on line 25 that makes use of it… so technically this parameter will currently be unused and ignored by the action's implementation 😓

Reply:
Ahh yeah, thanks for catching that! I completely missed adding the rest of the implementation. I've added it in a2c79346b43c0187f8f1f166941dc99808d32fdd